packages/builder/src/components/automation/SetupPanel/AutomationBlockSetup.svelte

```diff
@@ -356,7 +357,8 @@
       value.customType !== "queryParams" &&
       value.customType !== "cron" &&
       value.customType !== "triggerSchema" &&
-      value.customType !== "automationFields"
+      value.customType !== "automationFields" &&
+      value.type !== "attachment"
```

Comment:
Should we handle the single attachment in here?

Reply:
This is just an AutomationIOType, not related to actual Budibase Types